plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarLogic.java

```diff
@@ -171,7 +173,7 @@ public String getEventsTomorrow() {
 			zdt = endOfTomorrow.atZone(ZoneId.systemDefault());
 			DateTime max = new DateTime(zdt.toInstant().toEpochMilli());
 			Events events = this.calendarService.getService().events().list(this.primary).setTimeMin(setup)
-					.setTimeMax(max).setOrderBy(this.orderBy).setSingleEvents(true).execute();
+					.setTimeMax(max).setOrderBy(this.orderBy).setSingleEvents(Boolean.valueOf(true)).execute();
```

Comment:
removed Boolean.valueOf

Reply:
I thought because of the boxed warning that this may be a good solution.